elfi/methods/bo/acquisition.py

```diff
@@ -172,13 +172,10 @@ def evaluate(self, x, t=None):
         t : int
             Current iteration (starting from 0).
         """
-        if not isinstance(t, int):
-            raise ValueError("Parameter 't' should be an integer.")
```

Comment:
I think this check is good so that e.g. None is not accepted. Maybe
```
try:
    t = int(t)
except:
    raise ValueError("Parameter 't' should be an integer.")
```